lib/internal/test_runner/reporter/spec.js

```diff
@@ -60,12 +62,26 @@ class SpecReporter extends Transform {
       ), `\n${indent}  `);
     return `\n${indent}  ${message}\n`;
   }
-  #handleEvent({ type, data }) {
+  #formatTestReport(type, data, prefix = '', indent = '', hasChildren = false, skippedSubtest = false) {
     let color = colors[type] ?? white;
     let symbol = symbols[type] ?? ' ';
-
+    const duration_ms = data.details?.duration_ms ? ` ${gray}(${data.details.duration_ms}ms)${white}` : '';
+    const title = `${data.name}${duration_ms}${skippedSubtest ? ' # SKIP' : ''}`;
+    if (hasChildren) {
+      // If this test has had children - it was already reported, so slightly modify the output
+      return `${prefix}${indent}${color}${symbols['arrow:right']}${white}${title}\n`;
+    }
+    const error = this.#formatError(data.details?.error, indent);
+    if (skippedSubtest) {
+      color = gray;
+      symbol = symbols['hyphen:minus'];
+    }
+    return `${prefix}${indent}${color}${symbol}${title}${error}${white}`;
```

Comment:
```suggestion
    return `${prefix}${indent}${color}${symbol}${title}${white}${error}`;
```
this is the fix for https://github.com/nodejs/node/pull/47164#issuecomment-1477646648
since it is a small fix, I think it would be ok to include it in this PR

Reply:
Shall we add a test for this as well? In `test/pseudo-tty/test_runner_default_reporter.out`.